server/routes/idea.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const idea = require('../models/idea.js');
+const router = express.Router();
+
+//index
+router.get('/', function(req, res) {
+	idea.all(function(value){
+		for(i=0; i < value.length; i++){
+			delete value[i]["_id"];
+		}
+		console.log(value);
+		res.json(value);
+	});
+});
+
+//show
+router.get('/:id', function(req, res) {
+	idea.get(req.params.id, function(value){
+
+		//If no idea with :id exists
+		if(value.length == 0)
+			res.json({success : false});
+		else{
+			value = value[0]; //extract object from array
```

Comment:
If `idea.get` is always returning a single idea, it might be a good idea to use the `collection.findOne` method instead.